src/staking-router-modules/contracts/staking-router/staking-router-fetch.service.ts

```diff
@@ -1,26 +1,30 @@
 import { Injectable, Inject, LoggerService } from '@nestjs/common';
-import { StakingModule } from './staking-module';
+import { StakingModule } from '../../interfaces/staking-module.interface';
+import { STAKING_MODULE_TYPE } from '../../constants';
 import { LOGGER_PROVIDER } from '@lido-nestjs/logger';
-import { IStakingModuleService } from 'common/contracts/i-staking-module';
-import { STAKING_MODULE_TYPE } from 'staking-router-modules';
-import { LidoLocatorService } from 'common/contracts/lido-locator';
-import { StakingRouter__factory } from 'generated';
-import { ExecutionProvider } from 'common/execution-provider';
+import { StakingModuleInterfaceService } from '../staking-module-interface';
+import { LidoLocatorService } from '../lido-locator';
+// TODO: maybe we need to move generated to the same folder with contracts
+import { StakingRouter__factory } from '../../../generated';
+// TODO: maybe ../../../ shows us that we need to move execution-provider on level up
+import { ExecutionProvider } from '../../../common/execution-provider';
 import { BlockTag } from '../interfaces';
-import { Trace } from 'common/decorators/trace';
+// TODO: maybe it is time to stop using trace decorators in code and use it only in debug mode
+import { Trace } from '../../../common/decorators/trace';
 
 const TRACE_TIMEOUT = 30 * 1000;
 
 @Injectable()
 export class StakingRouterFetchService {
   constructor(
     @Inject(LOGGER_PROVIDER) protected readonly logger: LoggerService,
-    protected readonly iStakingModule: IStakingModuleService,
+    protected readonly stakingModuleInterface: StakingModuleInterfaceService,
```

Comment:
maybe `commonStakingModule`? I'm confused a bit by the word `interface` in the class instance